src/vmm/src/builder.rs

```diff
@@ -884,49 +884,38 @@ fn attach_block_devices<'a, I: Iterator<Item = &'a BlockDeviceType> + Debug>(
     blocks: I,
     event_manager: &mut EventManager,
 ) -> Result<(), StartMicrovmError> {
+    macro_rules! attach_block {
```

Comment:
commit message: s/dublication/duplication/

I have an intuition that this extraction can be done without using a macro. It would be a bit easier to read if we manage to do so.

Reply:
Discussed offline, there is no simpler way to do this because block have different types and we need to access fields of these types directly.

Reply:
the commit message typos are still present in `refactor: removed dublication from block device attachment`